### ISSUE TITLE
[Enhancement] Rounding by unit/physical quantity

### ISSUE THREAD

**Issue by emanuelemaz** [Proektsoftbg/Calcpad#145]
It could be useful to round to different digits based on the unit or the physicial quantity used.
For example, I do some reports that require 3 digits for distances and 4 digits for angles. I usually round to 4 digits and then correct the distances on MS Word, but an output control (e.g. `#round 3 m` or `round 3 length`) could be a better way to handle this.

**Comment by Proektsoftbg** [comment]
Yes, it really sounds like a good idea. I never had such a strict requirements about rounding, but I can really understand the situation when it can be needed. 

But, I can also see that you have good programming skills. :)

This project is growing and getting more and more demanding, but my time is limited, since I also have another job. So, we are starting to need contributors that can help on coding, documentation, etc. If you are willing to try, this issue seams to be a good chance for that, and I can help of course, if needed. It is not very difficult to implement. :)

Also there are other tasks that are much needed, and easy for every programmer, but I still do not have time to complete. Such are: writing unit tests, making the app multilingual and localizing for different languages, etc. 

It is difficult for one person to do everything, and also very slow. But if more people are involved, together we can make it much better for much shorter time. So, anyone that is willing to contribute is quite welcome. Calcpad seems to be a very promising project already. So, I started working on vectors and matrices, so it will be soon a full featured math system for engineers.

**Comment by emanuelemaz** [comment]
Sure, I'll see what I can do!
BTW, congratulations for the work: I think Calcpad already is the best software for hand calculations, being so versatile, free and open source. It handles units the best among its competitors. I've tried MathCAD, Jupyter with handcalcs, SMath Studio, but this is the best. I'll be happy to contribute if I manage to!

**Comment by Proektsoftbg** [comment]
Thank you for you kind words! If you can, your contribution would be highly appreciated, but if you can't, it will not be a trouble at all. You will still be quite welcomed to the Calcpad community. :)

In any case, I added your suggestion to the wish list and it will be implemented. It just may take a bit longer.
